src/Sylius/Component/Sequence/Repository/NumberListenerInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Sequence\Repository;
+
+use Doctrine\ORM\Event\PreFlushEventArgs;
+use Sylius\Component\Sequence\Model\SequenceSubjectInterface;
+use Sylius\Component\Sequence\Registry\NonExistingGeneratorException;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+interface NumberListenerInterface
```

Comment:
IMHO we shouldn't create interfaces for listeners, listener should only execute related service's method on given event.